rdp-server/icp/pbrpc/pbrpc.c

```diff
@@ -190,7 +190,7 @@ static int pbrpc_process_response(pbRPCContext* context, Ogon__Pbrpc__RPCBase *r
 static int pbrpc_process_message_out(pbRPCContext* context, Ogon__Pbrpc__RPCBase *msg)
 {
 	int ret;
-	char msgLen = ogon__pbrpc__rpcbase__get_packed_size(msg);
+	int msgLen = ogon__pbrpc__rpcbase__get_packed_size(msg);
```

Comment:
wouldn´t `size_t` or `ssize_t` be the better match for sizes passed to `malloc`?

Reply:
That's what I did as first intention, and then I get the burden of sign/unsigned comparison later. And protobuf function always return positive values (except when pack failed which is more an assert than a real case).